src/sync.cpp

```diff
@@ -871,12 +891,6 @@ LocalNode* Sync::checkpath(LocalNode* l, string* localpath, string* localname)
     }
     else
     {
-        if (initializing)
-        {
-            delete fa;
-            return NULL;
-        }
-
         LOG_warn << "Error opening file";
```

Comment:
With this change, we keep returning NULL, but now we'll do also some actions that we used to not do when we were initializing. Not sure if that's your aim.